ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -695,8 +726,8 @@
         removed? (and last-route-change? (max-date-within-evaluation-window? route))
         no-traffic? (and no-traffic-start-date
                          (or no-traffic-end-date
-                             (and (> no-traffic-run no-traffic-detection-threshold)
-                                  (.isAfter no-traffic-start-date (.toLocalDate (:max-date route))))))
+                             (and (pos? no-traffic-run)
+                                  (.isBefore no-traffic-start-date (.toLocalDate (:max-date route))))))
```

Comment:
Did you leave deliberately threshold compare out ?

Reply:
Yes indeed, `detect-no-traffic-run` reports only long enough runs. 
The removed check here prevented reporting some of the nil-traffic day changes which come from the new daily comparison which should report all new traffic change days.